jobs/bump-lockfile.Jenkinsfile

```diff
@@ -77,44 +112,70 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
       done
     """)
 
-    if (!timestampOnly) {
-        stage("Fetch") {
-            // XXX: hack around subtle lockfile bug (jlebon to submit an
-            // rpm-ostree issue or patch about this)
-            shwrap("cp src/config/fedora-coreos-pool.repo{,.bak}")
-            shwrap("echo cost=500 >> src/config/fedora-coreos-pool.repo")
-            shwrap("cosa fetch --strict")
-            shwrap("cp src/config/fedora-coreos-pool.repo{.bak,}")
-        }
+    // The bulk of the work (build, test, etc) is done in the following.
+    // We only need to do that work if we have changes.
+    if (haveChanges) {
+        parallel "Fetch/Build/Test aarch64": {
+            shwrap("""
+               cat <<'EOF' > spec.spec
+job:
+  strict: true
+  miniocfgfile: ""
+recipe:
+  git_ref: testing-devel
+  git_url: https://github.com/coreos/fedora-coreos-config
+stages:
+- id: ExecOrder 1 Stage
+  execution_order: 1
+  description: Stage 1 execution base
+  build_artifacts: [base]
+  post_commands:
+    - cosa kola run --basic-qemu-scenarios
```

Comment:
Is the idea here that eventually we'll add more tests to match the x86_64 path?

Reply:
Does this handle fetching back the kola test results so that we can archive them?

Reply:
All logs and Kola logs are specifically returned by Gangplank.

Reply:
There is code in gangplank to pull back kola logs. I'm sure we might need to tweak some things. Let's fix in a followup if that's OK.